backend/donations/migrations/0031_cause_allow_online_notifications_and_more.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 5.1.7 on 2025-04-04 12:28
```

Comment:
We need an extra migration to copy the organization's contact email to the notification email if the organization allows online signing of forms.